### PR TITLE
Fuse safe integration

### PR DESCRIPTION
**Pull request by t0mcr8se**
## TODO: 

- [X] Remove gnosis safe links in the footer
- [X] Remove intercom support message board
- [X] Change support links to fuseio telegram
- [X] Change safe-services api 
- [X] Delete other networks from src/config/networks
- [X] Fix exchange price issue
- [X] Change logo and color scheme

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,4 +8,4 @@ services:
       - /app/src/types/contracts
       - ./:/app
     ports:
-      - 3000:3000
+      - 3002:3000
```
Comment (leonprou) [inline docker-compose.yml]: can you change this back?

File: package.json [hunk 2]
```diff
@@ -181,7 +181,7 @@
     "abi-decoder": "^2.4.0",
     "axios": "0.21.4",
     "bignumber.js": "9.0.1",
-    "bnc-onboard": "~1.35.3",
+    "@t0mcr8se/bnc-onboard": "1.35.6",
```
Comment (leonprou) [inline package.json]: can you transfer the repo to fuseio and use our repo here?
I suppose I would need to publish this

File: package.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "build-desktop": "yarn compile-electron && cross-env REACT_APP_BUILD_FOR_DESKTOP=true REACT_APP_ENV=production yarn build-mainnet",
-    "build": "cross-env REACT_APP_APP_VERSION=$npm_package_version rescripts --max-old-space-size=8192 build",
+    "build": "cross-env PUBLIC_URL=/ REACT_APP_APP_VERSION=$npm_package_version rescripts --max-old-space-size=8192 build",
```
Comment (leonprou) [inline package.json]: no need to change this here, I handle this in the codebuild for S3 only

File: src/config/__tests__/config.test.ts [hunk 4]
```diff
@@ -1,122 +1,122 @@
...
-    const TX_SERVICE_URL = xdai.environment.dev?.txServiceUrl
-
-    // When
-    const txServiceUrl = getTxServiceUrl()
-
-    // Then
-    expect(TX_SERVICE_URL).toBe(txServiceUrl)
-  })
-})
+// const { mainnet, xdai } = networks
```
Comment (leonprou) [inline src/config/__tests__/config.test.ts]: Can you make this work with fuse and fusespark?
Any issues this should not work

**Review by leonprou — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (t0mcr8se) [reply package.json]: https://github.com/fuseio/onboard
this repo is ready to be deployed to npm, let me know when you deploy it so I can make the changes

File: src/config/__tests__/config.test.ts (hunk 4 above)
Comment (t0mcr8se) [reply src/config/__tests__/config.test.ts]: done

File: src/routes/CreateSafePage/steps/NameNewSafeStep.tsx [hunk 5]
```diff
@@ -63,7 +63,7 @@ function NameNewSafeStep(): ReactElement {
           <Link href="https://gnosis-safe.io/privacy" rel="noopener noreferrer" target="_blank">
             privacy policy
           </Link>
-          . Most importantly, you confirm that your funds are held securely in the Gnosis Safe, a smart contract on the
+          . Most importantly, you confirm that your funds are held securely in the Fuse Safe, a smart contract on the
           Ethereum blockchain. These funds cannot be accessed by Gnosis at any point.
```
Comment (leonprou) [inline src/routes/CreateSafePage/steps/NameNewSafeStep.tsx:67]: Ethereum blockchain  -> Fuse Network

File: package.json [hunk 6]
```diff
@@ -20,7 +20,7 @@
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "build-desktop": "yarn compile-electron && cross-env REACT_APP_BUILD_FOR_DESKTOP=true REACT_APP_ENV=production yarn build-mainnet",
-    "build": "cross-env REACT_APP_APP_VERSION=$npm_package_version rescripts --max-old-space-size=8192 build",
+    "build": "cross-env REACT_APP_APP_VERSION=$npm_package_version rescripts  --max-old-space-size=8192 build",
```
Comment (leonprou) [inline package.json:23]: please delete extra space :P

**Review by leonprou — APPROVED** [review]

**Pull request merged**